app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index; end
+
+  def show
+    @post = Post.find params[:id]
+  end
+
+  def create
+    @post = Post.new(message: params[:msg])
+    @post.save
```

Comment:
What happens if the save fails? It would be a good idea to check the result of the save and display an error message if it fails.